receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -203,18 +203,24 @@ metrics:
       value_type: int
     attributes: []
     enabled: true
+  # The calculation for node.disk.io.read is actually in KiBy(1024 bytes), not Kby (1000 bytes)
+  # Source: https://github.com/elastic/elasticsearch/blob/3c6797f2d2271a30b24f505da55afbb5ea10733e/server/src/main/java/org/elasticsearch/monitor/fs/FsInfo.java#L287
+  # The calculation uses uses IoStats package where blocks are equivalent to sectors that have the size of 512 bytes https://linux.die.net/man/1/iostat
```

Comment:
I need you to connect some more dots for me because I am not able to follow how these links indicate what you are claiming.

Reply:
added a more detailed explanation with sources